x-pack/plugins/spaces/public/plugin.test.ts

```diff
@@ -236,129 +235,34 @@ describe('Spaces plugin', () => {
   });
 
   describe('isSolutionViewEnabled', () => {
-    it('when onCloud, not serverless and allowSolutionVisibility is "true"', () => {
+    it('when allowSolutionVisibility is "true"', () => {
```

Comment:
Question: should there also be a test for when `allowSolutionVisibility` is `undefined`?

Reply:
It can't be undefined. It comes from the server yml config that has a default to `true`.

Reply:
With that said, I added a test just in case the server would not provide a value
https://github.com/elastic/kibana/pull/203239/commits/2fb0eec4b487c40f829cd655ed56572c60f19645